charter.md

```diff
@@ -36,6 +36,17 @@
   ecosystems in a common way; they look towards the IETF for
   developing SDF 1.0 further into a high-quality specification.
 
+  The IETF also defines the YANG data modelling language (RFC 7950).
+  YANG has a strong focus on modelling the management interface for network devices,
+  using a fairly small set of network management protocols,
+  whereas SDF is designed to span (and transitively unify) many existing interaction models already present.
+  While YANG can be serialized to a variety of wire formats (XML, JSON, CBOR, ...),
+  the serialization is driven directly by the structure of the YANG.
```

Comment:
, there is no separate "protocol binding" step as in the use of SDF.

Reply:
I don't know how to process this. Is it a suggestion?

Reply:
Well, you just add the text after "the structure of the YANG" (or what you now made of that).